manuscript/manuscript.Rmd

```diff
@@ -112,11 +111,36 @@ Rich, processes-based simulations such as the SORTIE model in forest management
+While the history of improved efficiency in computing technology has shown a remarkable ability to reduce such barriers, it has simultaneously moved the leading edge of those capabilities farther beyond reach of traditional ecological research. 
+We believe that ecologists must seek to better understand the design, capabilities and limitations of these algorithms, even though their application will surely require the ambitious collaboration, resources and expertise on par with the scale of the immense environmental and ecological problems we face.
+
 In this paper, we draw on examples from fisheries management and ecological tipping points to illustrate how deep RL techniques can successfully discover optimal solutions to previously solved management scenarios and discover highly effective solutions to unsolved problems.
-We demonstrate that RL-based approaches are capable but by no means a magic bullet: reasonable solutions require careful design of training environments, choice of RL algorithms, tuning and evaluation, as well as substantial computational power.
-Our examples are intentionally simple, aiming to provide a clear template for understanding that could be easily extended to cover more realistic conditions. 
+We focus on examining the potential and limitations of deep RL through the lens of simple, classical models.
+Over a century of theory and practice in ecology has demonstrated that simple models can provide meaningful insights which improve management outcomes [@Getz2017].
+As Richard Levins successfully established in his classic paper on the principles of model building [@Levins1966], model complexity must not be mistaken for model realism.
+Levins esposed simple mechanistic models which satisfy the goals of being both _realistic_ and _general_. 
+More complex models such as those used in fisheries to guide the management of specific stocks typically sacrifice _generality_ for _precision_, a distinction Bob May characterized as "tactical models" (precise) rather than "strategic models" (general). 
+Such simple, realistic and general models are still the bedrock of most theory and practice today, even when implementation turns to tactical models for quantitative guidance (for instance, the notion of maximum sustainable yield, MSY, in fisheries, or $R_0$ in epidemiology, remain important concepts in management even though they do not exist in typical tactical models).
+These models provide an ideal first benchmark for evaluating the performance of emerging methods of deep RL for several reasons:
+Firstly, for some cases the optimal solution is already known, providing a clear standard-of-comparison to evaluate RL performance. 
+Prior work sometimes overlooks this essential step, assuming that whatever behavior an RL agent produces is sufficiently optimal or otherwise an accurate proxy for behavior arising from animal learning or evolution. 
```

Comment:
Seems like we should cite this sentence.

Reply:
yeah, kinda begs for citation, though hesitant to call anyone out specifically.  Maybe we can just cite the gym paper on the generic importance of benchmarks?

Reply:
Hmm thinking more about the citation here. I feel like the animal behavior part might come off as confusing to someone who isn't already familiar with the behavioral rl framing. I think we could drop the behavior part and just cite any algorithm benchmarking paper to cover the "rl agent as sufficiently optimal" assumption here.

Reply:
agree on the idea of a more general citation to benchmarking.  can you drop something in here?